### PR TITLE
Add installudevrules command for linux

### PR DESCRIPTION
**Pull request by lontivero**
This PR is for discussing the concept and the implementation. The idea is to make it easier for Linux users to install the udev rules in their systems by running:

```
$ hwi installudevrules
```

### CODE REVIEW

**Comment by achow101** [comment]
Would it be possible to add a test for this? Maybe add an argument to the command that specifies the location so for tests install the rules to that location and check the files are correct?

File: hwilib/udevinstaller.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            udev_installer = UDevInstaller()
+            udev_installer.copy_udev_rule_files(location)
+            udev_installer.trigger()
+            udev_installer.reload_rules()
+            udev_installer.add_user_plugdev_group()
+        except CalledProcessError as e:
+            if geteuid() != 0:
+                return {'error':'Need to be root.','code':NEED_TO_BE_ROOT}
+            raise
+        return {}
```
Comment (achow101) [inline hwilib/udevinstaller.py]: This should return `{"success": True}` instead of an empty obj.

**Comment by lontivero** [comment]
@achow101 I had to duplicate code for error handling. Do you want I create a error handling function like this: https://github.com/bitcoin-core/HWI/pull/181#issuecomment-495306348?

**Comment by achow101** [comment]
> I had to duplicate code for error handling. Do you want I create a error handling function

I think this is fine for now. I plan on making an error handling pr soon which can take care of this.

**Comment by achow101** [comment]
Please squash

**Comment by achow101** [comment]
ACK cada7f5ce0cf0854d7fc526b366107d5f0d868aa

**Pull request merged**